web/src/SidebarResources.tsx

```diff
@@ -366,18 +342,6 @@ export class SidebarResources extends React.Component<SidebarProps> {
       </SidebarResourcesRoot>
     )
   }
-
-  render() {
-    return (
-      <PersistentStateProvider
-        defaultValue={defaultOptions}
```

Comment:
👐 PersistentStateProvider was kind of awkward and I'm pretty sure it should have been called a Consumer, not a Provider!